client/app/pipes/pluck-keys.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({name: 'pluckKeys'})
+export class PluckKeysPipe implements PipeTransform {
+  transform(value: any, args: any[]): any {
+    if(!value) return value;
+
+    return Object.keys(value.rows);
```

Comment:
### Critical

The idea for pipes is to create re-usable chunks of worker functions, however this is tightly coupled to it's use in the Accordion Component, because it expects all arguments passed to have a `rows` property. Please refactor this to make it more loosely coupled.